ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+> Nuestros eventos se rigen por el siguiente [Código de conducta](https://github.com/Noders/code_of_conduct) - [https://github.com/Noders/code_of_conduct](https://github.com/Noders/code_of_conduct) 
+
+> Antes de suscribir una charla, recuerda revísarlo para ver si estamos en la misma página 😊
```

Comment:
Nitpick/Non-blocker: Esta frase suena un poco rara en español. Es traducción literal the "on the same page"?